code_schemes/s01e01.json

```diff
@@ -147,6 +147,230 @@
         "CodeType": "Normal",
         "StringValue": "no_other_theme"
       },
+      {
+        "CodeID": "code-a3578f40",
+        "CodeType": "Normal",
+        "DisplayText": "about coronavirus",
+        "NumericValue": 14,
+        "StringValue": "about_coronavirus",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-798f977f",
+        "CodeType": "Normal",
+        "DisplayText": "symptoms",
+        "NumericValue": 15,
+        "StringValue": "symptoms",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-d606d1e8",
+        "CodeType": "Normal",
+        "DisplayText": "spread/transmission",
+        "NumericValue": 16,
+        "StringValue": "spread_transmission",
```

Comment:
I think this was called how spread/transmitted elsewhere, not sure if you need to change here.

Reply:
adding it as is in the thematic document.